amanuensis/blueprints/project.py

```diff
@@ -119,6 +119,7 @@ def get_projetcs():
         statuses_by_consortium = set()
         consortiums = set()
         for request in project["requests"]:
+            #TODO this isnt always ordering them correctly
```

Comment:
What is the issue here?

In the worst case you can use the other function you wrote (get_latest_request_state_by_id). If you can get it ordered without extra calls tho it would be great.

Reply:
right now this is the stack of states for a request'states': [
        {'id': 1, 'name': 'In Review', 'code': 'IN_REVIEW', 'create_date': '2023-12-19T18:06:05.028079', 'update_date': '2023-12-19T18:06:05.028079'}, 
        {'id': 3, 'name': 'Approved', 'code': 'APPROVED', 'create_date': '2023-12-19T18:06:05.028079', 'update_date': '2023-12-19T18:06:05.028079'}, 
        {'id': 14, 'name': 'Data Downloaded', 'code': 'DATA_DOWNLOADED', 'create_date': '2023-12-19T18:06:05.028079', 'update_date': '2023-12-19T18:06:05.028079'}, 
        {'id': 4, 'name': 'Data Available', 'code': 'DATA_AVAILABLE', 'create_date': '2023-12-19T18:06:05.028079', 'update_date': '2023-12-19T18:06:05.028079'}
    ],  and the states were changed in the correct order, I will look into the project schema to see if I can get access to the request_state create times